.github/workflows/performance-test.yml

```
@@ -35,3 +35,13 @@ jobs:
       working-directory: operator
       run: |
         make performance-test
+    
+    - name: Create reports
+      working-directory: ./.github/workflows
+      run: ./create_reports.sh
+
+    - name: Upload cluster snapshot 
+      uses: actions/upload-artifact@v3
```

You need to add `if: always()` here to tell github actions to also run those steps when the previous tests failed
```suggestion
    - name: Create reports
      if: always()
      working-directory: ./.github/workflows
      run: ./create_reports.sh

    - name: Upload cluster snapshot
      if: always()
      uses: actions/upload-artifact@v3
```